data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/schema/ErrorListBadRequest.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2022 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.beaconrestapi.schema;
+
+import static tech.pegasys.teku.infrastructure.http.HttpStatusCodes.SC_BAD_REQUEST;
+
+import java.util.List;
+import java.util.Objects;
+import tech.pegasys.teku.validator.api.SubmitDataError;
+
+public class ErrorListBadRequest {
+  private static final int CODE = SC_BAD_REQUEST;
+  private final String message;
+  private final List<SubmitDataError> errors;
+
+  public ErrorListBadRequest(final String message, final List<SubmitDataError> errors) {
+    this.message = message;
+    this.errors = errors;
```

Comment:
this looks the same as the PostSyncCommittees response, can we use the same classes?

Reply:
Yes, should I do this in a seperate PR?

Reply:
can probably just use whats already there without another pr? what would you prefer?

Reply:
Okay, I'll do it in this one

Reply:
Actually would be cleaner to do in another one, so will do that after this has merged